lib/chef/dist.rb

```diff
@@ -15,5 +15,8 @@ class Dist
 
     # product website address
     WEBSITE = "https://chef.io".freeze
+
+    # The key that mixlib-install uses to refer to Chef Infra packages
+    MIXLIB = "chef".freeze
```

Comment:
This seems like premature optimization here. We'd only be using it in licensing checks which the community edition wouldn't care about anyways.

Reply:
I'm happy to hardcode it back to `"chef"` for now instead of referencing the DIST